godtools/App/Share/Data/MobileContentAuthTokenRepository/Api/Models/MobileContentAuthTokenDataModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MobileContentAuthTokenDataModel.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 11/1/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+struct MobileContentAuthTokenDataModel: Decodable {
+    
+    let token: String
+    let expirationDate: String
+    let userId: Int
+    
+    enum RootKeys: String, CodingKey {
+        case data
+    }
+    
+    enum DataKeys: String, CodingKey {
+        case attributes
+    }
+    
+    enum AttributesKeys: String, CodingKey {
+        case token
+        case expiration
+        case userId = "user-id"
+    }
+    
+    init(from decoder: Decoder) throws {
+        
+        let container = try decoder.container(keyedBy: RootKeys.self)
+        
+        let dataContainer = try container.nestedContainer(keyedBy: DataKeys.self, forKey: .data)
+        let attributesContainer = try dataContainer.nestedContainer(keyedBy: AttributesKeys.self, forKey: .attributes)
+        
+        token = try attributesContainer.decode(String.self, forKey: .token)
+        expirationDate = try attributesContainer.decodeIfPresent(String.self, forKey: .expiration) ?? ""
```

Comment:
One more thing, I'm not sure if we will use the expirationDate, but it would be nice to have this decoded to a Date that way we have a Date type to work with if we need to do any sort of comparison or date checking.  ```ISO8601DateFormatter().date(from: <String>)```